Legacy/PlayFab/Admin.api.json

```diff
@@ -20079,10 +19859,6 @@
       "result": "UpdateCatalogItemsResult",
       "resultExample": "",
-      "seeAlso": [
```

Comment:
Can you file a bug to investigate why these seeAlso links are being removed? I see this go back and forth every now and then and don't know why.